lib/tasks/addresses.rake

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+namespace :address do
+  desc "Fix addresses set in the renewals process to match old data format"
+  task update_from_os_places: :environment do
+    registrations = WasteCarriersEngine::Registration.where(:past_registration.exists => true)
```

Comment:
I would suggest instantiating these within the methods themselves. What we have makes the methods themselves more flexible and easier to test, but there may be a performance hit.

We have to hold both collections in memory until both operations have completed. Moving the instantiating to the relevant methods moves the variable to a more local scope, and will allow the GC to get in and scrub it from memory if it needs to.